src/main/java/au/org/ala/biocache/dto/DownloadRequestParams.java

```diff
@@ -243,6 +259,8 @@ public String getFileType() {
         return fileType;
     }
 
+    public String getEmailTemplate() { return emailTemplate; }
```

Comment:
Put the return statement on its own line to match the rest of the style for biocache-service.

Reply:
I am just following the style on this file.

Reply:
I am not sure what you are referring to. The rest of this file has a style that is common with the rest of the biocache-service code.